docs/user-manual/modules/ROOT/pages/camel-upgrade-recipes-tool.adoc

```diff
@@ -0,0 +1,21 @@
+= Apache Camel Upgrade Recipes Project
+
+This document provides instructions on how to use the automatic update tool to assist in the migration process.
+
+== Introduction
+
+To help with migration process, Camel Upgrade Recipes project based on OpenRewrite was developed.
+
+NOTE:  The project is not designed to provide fully automated migration but rather to assist with manual migrations, making them more efficient and less error-prone.
+
+Refer to the https://github.com/apache/camel-upgrade-recipes/blob/main/release_notes.adoc[release notes] of the Camel Upgrade Recipes Project to see the specific migrations covered by the upgrade tool.
+
+== Running the Upgrade Tool
```

Comment:
The update tool can be run via Camel CLI as well, do you think it is worth mentioning it too? https://camel.apache.org/manual/camel-jbang.html#_update

Reply:
I'd say that we should mention it here as well as another option.

Reply:
I updated the PR with camel-jbang update option